### PR TITLE
Relax the wild dep

### PR DESCRIPTION
**Pull request by sylvestre**
to make life easier of distro packagers.

### CODE REVIEW

**Comment by rivy** [comment]
I submitted a [request](https://gitlab.com/kornelski/wild/-/merge_requests/3) to [`wild`](https://gitlab.com/kornelski/wild) requesting a minor change which would return v1.31.0 compatibility. We could relax the versioning if that request is merged.

**Comment by sylvestre** [comment]
The new version of wild that @rivy improved has been published. We are good

File: Cargo.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ members=["src/uucore_procs"]
 [dependencies]
 dunce = "1.0.0"
 getopts = "<= 0.2.21"
-wild = "2.0.0, < 2.0.2"
+wild = "2.0"
```
Comment (rivy) [inline Cargo.toml]:
```suggestion
wild = "2.0.4"
```

**Review by rivy — CHANGES_REQUESTED** [review]
It should pass with the suggested dependency version.

After the change, I'll push it to *canary* to allow for CI testing.

**Comment by rivy** [comment]
Just FYI, `wild = "2.0.4"` is a fairly loose specification, only differing from `wild = "2.0"` by requiring greater than or equal to `2.0.4`.

`wild = "2.0.4"` == `wild = "^2.0.4"` == `wild = ">= 2.0.4, <3.0.0"` (per [cargo docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html)).

**Pull request merged**